### PR TITLE
translate spec/cloudevents/languages/zh-CN/adapters/aws-s3.md to zh-cn

### PR DESCRIPTION
**Pull request by VedaGao**
translate spec/cloudevents/languages/zh-CN/adapters/aws-s3.md to zh-cn

Signed-off-by: VedaGao <gymti2014@163.com>

### CODE REVIEW

**Comment by VedaGao** [comment]
@JieDing Please review this translation, thank you.

File: cloudevents/languages/zh-CN/adapters/aws-s3.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+| `specversion`         | `1.0`                                                        |
+| `type`                | `com.amazonaws.s3.` + "eventName" value                      |
+| `datacontenttype`     | S3 event type (e.g. `application/json`)                      |
+| `dataschema`          | Omit                                                         |
+| `subject`             | "s3.object.key" value                                        |
+| `time`                | "eventTime" value                                            |
+| `data`                | S3 event                                                     |
+
+注释：
+- 当事件来自 S3 时，“eventSource”值将始终为静态 (`aws:s3`)，但如果其他云厂商支持 S3 事件格式，则该值可能不会是 `aws:s3` - 而是特定于他们的环境的东西。
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/adapters/aws-s3.md:23]:
```suggestion
- 当事件来自 S3 时，“eventSource”将始终为固定值 (`aws:s3`)，但如果其他云厂商支持 S3 事件格式，则该值可能不会是 `aws:s3` - 而是特定于他们的环境的东西。
```

File: cloudevents/languages/zh-CN/adapters/aws-s3.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
-本文档尚未被翻译，请先阅读英文[原版文档](../../../adapters/aws-s3.md) 。
+本文档介绍如何将 [AWS S3 events](https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html) 事件转换为 CloudEvents。
 
-如果您迫切地需要此文档的中文翻译，请[提交一个issue](https://github.com/cloudevents/spec/issues) ，
-我们会尽快安排专人进行翻译。
\ No newline at end of file
+AWS S3 事件文档：
+https://docs.aws.amazon.com/AmazonS3/latest/dev/notification-content-structure.html
+
+所有 S3 事件都使用下表中描述的相同模式转换为 CloudEvents：
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/adapters/aws-s3.md:8]:
```suggestion
所有 S3 事件都通过下表中描述的模式转换为 CloudEvents：
```

**Review by JieDing — COMMENTED** [review]
Thx for your efforts. Here are some feedbacks. @VedaGao

**Comment by JieDing** [comment]
@VedaGao Thanks for your efforts again.

@duglin LGTM

**Pull request merged**